Library/Formula/libu2f-server.rb

```diff
@@ -0,0 +1,26 @@
+class Libu2fServer < Formula
+  desc "Yubico Universal 2nd Factor (U2F) Server C Library"
+  homepage "https://developers.yubico.com/libu2f-server/"
+  url "https://developers.yubico.com/libu2f-server/Releases/libu2f-server-1.0.1.tar.xz"
+  sha256 "a618f59051209d6d70c24cf42d64c9b67bd7dd5946b6dbd2c649181d7e8f1f6e"
+
+  depends_on "check" => :build
+  depends_on "json-c"
+  depends_on "openssl"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    ENV["LIBSSL_LIBS"] = "-lssl -lcrypto -lz"
+    ENV["LIBCRYPTO_LIBS"] = "-lcrypto -lz"
+
+    ENV["HELP2MAN"] = "/usr/local/bin/help2man --no-discard-stderr"
+    ENV["PKG_CONFIG"] = "/usr/local/bin/pkg-config"
+
+    system "./configure", *args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/u2f-server", "--version"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!